### DISCUSSION TITLE
ibax cuzdanı kurtarma yontemı kesın çözüm

### DISCUSSION THREAD

**Discussion by yasin001yk** [IBAX-io/go-ibax#54] [Ideas] [answered]
arkadaşlar cuzdanla alakalı sorun yasayanlar bu dedıgım yolu ızlerlerse herkes cuzdanını kurtarabılır öncelikle yapmanız gereken çok basıt  cuzdanı kurarken verılen kurtarma kelimeleri elınızde olması gerekıyor ÖNCELIKLE YENI CUZDAN KUR KISMINA TIKLIYORUZ DAHA ONCE VERILEN KELIMELERI BURAYA YAPIŞTIRIYORUZ ESKI ŞİFREMİZİ TEKRAR GİRİYORUZ AYNI SEYLERİ İKİNCİ KEZ YINE YAPIP KURTARMA KELİMELERİ VE SİFREYI GİRİYORUZ bukadar basıt eskısı gıbı şifrenızı tekrar girip giriş yapabılırsınız private key olmadan kurtarma kelimleri yle cuzdanınızı kurtarma yontemı ben uyguladım bı kaç arkadas daha uyguladı oldu bu bol şans herkese...

**Comment by HalloKyle** [comment, marked as answer]
Hi you can also check this url for solution: https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by hemresenol** [comment]
Doğru çözüm tebrikler 🎊
